backend/src/entities/book.ts

```diff
@@ -15,15 +15,15 @@ export class Book {
   @Column()
   subject!: string;
 
-  @Column({ nullable: true })
-  charName!: string;
-
-  @Column({ nullable: true })
-  charDesc!: string;
+  @Column({ nullable: true, length: 500 })
+  characters!: string;
 
```

Comment:
Column type should be JSON

Reply:
can you check the create book controller  I am storing them as one string.